src/libComponents/PasswordField/index.test.js

```diff
@@ -171,4 +171,28 @@ describe('PasswordField Component', () => {
 
     expect(input.props.secureTextEntry).toBe(true)
   })
+
+  test('passes inputTestID to password input', () => {
+    const { getByTestId } = render(
+      <ThemeProvider>
```

Comment:
I've noticed every test includes theme provider, maybe it's good idea to create some helper like `renderTheme()` and use it instead of `render(<PasswordField {...props}>)`